scst-store/logs.hbs.md

```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
+| logger   | string  | all             | Used to identify what produced the log entry. For Store, the name always starts with `MetadataStore`. For log entries that display the raw SQL queries, the name is `MetadataStore.gorm`     |
+| msg      | string  | all             | A short description of the logged event.                                                                                                                                                     |
+| hostname | string  | all             | The Kubernetes hostname of the pod handling the request. This helps identify the specific instance of the Store when you deploy multiple instances on a cluster.                             |
+| error    | string  | all             | The error message which is only available in error log entries.                                                                                                                              |
+| code     | integer | default         | The HTTP response code.                                                                                                                                                                      |
+| response | string  | default         | The HTTP response in human-readable format. For example, 'OK', 'Bad Request', or 'Internal Server Error'.                                                                                    |
+| query    | string  | debug           | The operation name will be key and value field will have the fields requested                                                                                                                                                                                                                             |
+
+#####  <a id='graphql-payload-out'></a> API payload log output
+
+As mentioned at the start of this section, by setting the verbosity level to `debug`, the AMR logs the
+body payload data for both the request and response of an API call.
+
+The `debug` verbosity level, instead of the `default`, is used to display this information instead of `default`
+because:
+
+* Body payloads can be huge and some information in these payloads may be sensitive, and the user may not want them exposed in
+production environment logs.
+
+Logs containing payload information may be in below mentioned format
+format:
+
+```console
+{"level":"info","ts":"2023-03-23T13:11:31.172966-06:00","logger":"Artifact Metadata Repository","msg":"Request response","hostname":"shaileshp2DMD6R.vmware.com","getAllApps":"query getAllApps {\n  apps(latest:true) {\n    \n    timestamp\n    location {\n      alias\n    }\n  }\n}","payload":{"apps":[{"timestamp":"2023-03-22T15:09:38.867371-06:00","location":{"alias":"1-Alias"}}]}}
+```
+
+##  <a id='slow_sql_query-out'></a> Slow SQL Query log output
+When verbosity level to `trace` you will see logs messages having some slow sql queries.
```

Comment:
Small reword, "When verbosity level is set to `trace` you will see log messages containing slow sql queries"